hardhat/hardhat.config.ts

```diff
@@ -132,8 +131,8 @@ task('create-user-policy', 'Creates a policy for the given user')
       userAddress,
       userAddress,
       parseUsd(args.coverageAmount),
-      BigNumber.from(Math.round(claimsAllowedFrom.getTime() / 1000)),
-      BigNumber.from(Math.round(claimsAllowedUntil.getTime() / 1000)),
+      Math.round(claimsAllowedFrom.getTime() / 1000),
```

Comment:
I guess this was just to reduce characters, right?

Reply:
Yeah it accepts `BigNumberish` arguments here, to the `BigNumber.from(...)` is unnecessary